src/pages/Login/KakaoLogin/KakaoLogin.js

```diff
@@ -0,0 +1,147 @@
+import React, { useState, useEffect } from 'react';
+import { API } from '../../../config';
+import styled from 'styled-components';
+
+const KakaoLogin = props => {
+  const { modalOpen, closeModal } = props;
+  const { Kakao } = window;
+  const { Login, setLogin } = useState(false);
+
+  const handleLogout = () => {
+    setLogin(!Login);
+    Kakao.API.request({
+      url: '/v1/user/unlink',
+      success: function (response) {
+        console.log(response);
+      },
+      fail: function (error) {
+        console.log(error);
+      },
+    });
+  };
+
+  const handleLogin = () => {
+    Kakao.Auth.login({
+      scope: 'profile_nickname, profile_image, birthday',
+      success: function (response) {
+        fetch(`${API.LOGIN}`, {
+          method: 'GET',
+          headers: {
+            Authorization: response.access_token,
+          },
+        })
+          .then(res => res.json())
+          .then(res => {
+            if (res) {
+              Kakao.Auth.logout();
+              localStorage.setItem('token', res);
+              console.log(res);
+            }
+          });
+
+        Kakao.API.request({
+          uri: '/v2/user/me',
+          success: function (response) {
+            console.log(response);
+          },
+          fail: function (error) {
+            alert('아이디와 비밀번호를 확인해주세요:)');
+          },
+        });
+        setLogin(!Login);
+      },
+      fail: function (error) {
+        alert('아이디와 비밀번호를 확인해주세요:)');
+      },
+    });
+  };
+
+  useEffect(() => {
+    if (Kakao.Auth.getAccessToken()) {
+    }
+  }, []);
+
+  return (
+    <>
+      {modalOpen ? (
+        <LoginWrapper>
+          <LoginHeader>
+            <ExitBtn onClick={closeModal}>×</ExitBtn>
+            <Title>로그인 또는 회원 가입</Title>
+          </LoginHeader>
+          <Greeting>
+            여행은 살아보고, <br /> 맥주는 마셔보는 거야! <br />
+            BeerBnB에 오신 것을 환영합니다.
+          </Greeting>
+          {!Login ? (
+            <KakaoButton
+              name="kakao"
+              onClick={handleLogin}
+              src="/images/kakao_login.png"
+              alt="login"
+            />
+          ) : (
+            <KakaoButton
+              name="kakao"
+              onClick={handleLogout}
+              src="/images/kakao_login.png"
+              alt="logout"
+            />
+          )}
```

Comment:
삼항연산자를 필요한 props 부분에만 적용해볼 수 있을 것 같습니다.

```suggestion
            <KakaoButton
              name="kakao"
              onClick={Login ? handleLogout : handleLogin}
              src="/images/kakao_login.png"
              alt={Login ? "logout" : "login"}
            />
```

Reply:
훨씬 깔끔해졌네요 감사합니다!